src/test/java/net/snowflake/client/jdbc/BindingDataLatestIT.java

```diff
@@ -75,7 +75,7 @@ public void testTimestampBindingWithNTZType() throws SQLException {
       // insert using stage binding
       PreparedStatement prepStatement =
           connection.prepareStatement("insert into stageinsert values (?,?,?,?)");
-      statement.execute("ALTER SESSION SET CLIENT_STAGE_ARRAY_BINDING_THRESHOLD = 1");
+      statement.execute("ALTER SESSION SET CLIENT_STAGE_ARRAY_BINDING_THRESHOLD = 5");
```

Comment:
interesting, would that alter query work? I got error of `invalid parameter` on my local test.

Reply:
Yes, we cannot run this test on our local machine. This is only valid in the Jenkins(?) testing.